hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/builtin/SqlPageCodec.java

```diff
@@ -147,6 +148,11 @@ public static void encode(ClientMessage clientMessage, SqlPage sqlPage) {
 
                     break;
 
+                case JSON:
+                    ListMultiFrameCodec.encodeContainsNullable(clientMessage, (Iterable<HazelcastJsonValue>) column, HazelcastJsonValueCodec::encode);
```

Comment:
We need to make sure that 5.0 clients don't receive this new type. Or discuss with the client guys what exactly needs to be done.